index.js

```diff
@@ -0,0 +1,102 @@
+const myLibrary = [
+  {
+    author: 'Jack', title: 'Title of book', pages: 400, hasRead: false,
+  },
+  {
+    author: 'Hoppie', title: 'Ma se poes', pages: 3, hasRead: false,
+  },
+  {
+    author: 'Mary', title: 'A book', pages: 200, hasRead: false,
+  },
```

Comment:
- [ ] myLibrary should contain instances of `Book` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/library/book_array_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/library))